src-java/portstate-topology/portstate-storm-topology/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     runtimeClasspath project(path: ':base-storm-topology', configuration: 'releaseArtifacts')
     implementation project(':kilda-reporting')
     implementation project(':network-messaging')
+    implementation project(':nbworker-messaging')
```

Comment:
If some object used outside specific topology (that owns messaging packet) should it be moved into a more "common/shared" messaging packet?